[bluetooth] Airthings stops updating, also not using refreshinterval

## Expected Behavior
Fetch updated values from Airthings Wave Plus device at the specified refreshinterval.

## Current Behavior
It's fetching values intermittently and at very short intervals, sometimes just seconds between. After some minutes it stops updating. The RSSI values continues to update. Image showing behaviour:
![image](https://user-images.githubusercontent.com/5099412/104917531-d1366680-5993-11eb-841f-0ffbc67db323.png)

openhab log displays: 
```
2021-01-15 13:06:09.448 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.bluetooth.airthings.internal.AirthingsWavePlusHandler@58fabbce': Duplicate channels bluetooth:airthings_wave_plus:adapter1:sensor1:rssi
java.lang.IllegalArgumentException: Duplicate channels bluetooth:airthings_wave_plus:adapter1:sensor1:rssi
        at org.eclipse.smarthome.core.thing.util.ThingHelper.ensureUniqueChannels(ThingHelper.java:165) ~[?:?]
        at org.eclipse.smarthome.core.thing.util.ThingHelper.ensureUniqueChannels(ThingHelper.java:157) ~[?:?]
        at org.eclipse.smarthome.core.thing.util.ThingHelper.ensureUniqueChannels(ThingHelper.java:153) ~[?:?]
        at org.eclipse.smarthome.core.thing.binding.builder.ThingBuilder.withChannel(ThingBuilder.java:76) ~[?:?]
        at org.openhab.binding.bluetooth.BeaconBluetoothHandler.initialize(BeaconBluetoothHandler.java:107) ~[?:?]
        at org.openhab.binding.bluetooth.airthings.internal.AirthingsWavePlusHandler.initialize(AirthingsWavePlusHandler.java:81) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor80.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_265]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_265]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:152) [bundleFile:?]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:52) [bundleFile:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_265]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_265]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_265]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
2021-01-15 13:06:09.448 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'bluetooth:airthings_wave_plus:adapter1:sensor1': Duplicate channels bluetooth:airthings_wave_plus:adapter1:sensor1:rssi
java.lang.IllegalArgumentException: Duplicate channels bluetooth:airthings_wave_plus:adapter1:sensor1:rssi
        at org.eclipse.smarthome.core.thing.util.ThingHelper.ensureUniqueChannels(ThingHelper.java:165) ~[?:?]
        at org.eclipse.smarthome.core.thing.util.ThingHelper.ensureUniqueChannels(ThingHelper.java:157) ~[?:?]
        at org.eclipse.smarthome.core.thing.util.ThingHelper.ensureUniqueChannels(ThingHelper.java:153) ~[?:?]
        at org.eclipse.smarthome.core.thing.binding.builder.ThingBuilder.withChannel(ThingBuilder.java:76) ~[?:?]
        at org.openhab.binding.bluetooth.BeaconBluetoothHandler.initialize(BeaconBluetoothHandler.java:107) ~[?:?]
        at org.openhab.binding.bluetooth.airthings.internal.AirthingsWavePlusHandler.initialize(AirthingsWavePlusHandler.java:81) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor80.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_265]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_265]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:152) [bundleFile:?]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:52) [bundleFile:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_265]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_265]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_265]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
```

This is my Bluetooth dongle thing:
`Bridge bluetooth:bluegiga:adapter1 [ port="COM3", backgroundDiscovery=false ]`

This is my Airthings thing:
`bluetooth:airthings_wave_plus:adapter1:sensor1 "Airthings Wave Plus Sensor 1" (bluetooth:bluegiga:adapter1) [ address="...", refreshInterval=600 ]`

[Related openHAB forum post](https://community.openhab.org/t/bluetooth-bluegiga-airthings-wave-plus-issue/114239)

## Possible Solution


## Steps to Reproduce (for Bugs)


## Context


## Your Environment
* Version used: openHAB 2.5.10, openHAB 2.5.11
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...): Windows 10, Zulu 8.48.0.53, jdk8.0.265

--- a/bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AbstractAirthingsHandler.java
+++ b/bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AbstractAirthingsHandler.java
@@ -115,24 +115,24 @@ abstract public class AbstractAirthingsHandler extends BeaconBluetoothHandler {
     }
 
     private synchronized void execute() {
-            ConnectionState connectionState = device.getConnectionState();
-            logger.debug("Device {} state is {}, serviceState {}, readState {}", address, connectionState, serviceState,
-                    readState);
+        ConnectionState connectionState = device.getConnectionState();
+        logger.debug("Device {} state is {}, serviceState {}, readState {}", address, connectionState, serviceState,
+                readState);
 
-            switch (connectionState) {
-                case DISCOVERING:
-                case DISCOVERED:
-                case DISCONNECTED:
-                    if (isTimeToRead()) {
-                        connect();
-                    }
-                    break;
-                case CONNECTED:
-                    read();
-                    break;
-                default:
-                    break;
-            }
+        switch (connectionState) {
+            case DISCOVERING:
+            case DISCOVERED:
+            case DISCONNECTED:
+                if (isTimeToRead()) {
+                    connect();
+                }
+                break;
+            case CONNECTED:
+                read();
+                break;
+            default:
+                break;
+        }
     }
 
     private void connect() {

--- a/bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AbstractAirthingsHandler.java
+++ b/bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AbstractAirthingsHandler.java
@@ -106,12 +106,15 @@ abstract public class AbstractAirthingsHandler extends BeaconBluetoothHandler {
     }
 
     private void executePeridioc() {
-        sinceLastReadSec.addAndGet(CHECK_PERIOD_SEC);
-        execute();
+        try {
+            sinceLastReadSec.addAndGet(CHECK_PERIOD_SEC);
+            execute();
+        } catch (Exception e) { // catch all to avoid scheduleWithFixedDelay being suppressed
+            logger.warn("Failed to read Airthings device", e);
+        }
     }
 
     private synchronized void execute() {
-        try {
             ConnectionState connectionState = device.getConnectionState();
             logger.debug("Device {} state is {}, serviceState {}, readState {}", address, connectionState, serviceState,
                     readState);
@@ -130,9 +133,6 @@ abstract public class AbstractAirthingsHandler extends BeaconBluetoothHandler {
                 default:
                     break;
             }
-        } catch (Exception e) { // catch all to avoid scheduleWithFixedDelay being suppressed
-            logger.warn("Failed to read Airthings device", e);
-        }
     }
 
     private void connect() {

--- a/bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AbstractAirthingsHandler.java
+++ b/bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AbstractAirthingsHandler.java
@@ -130,7 +130,7 @@ abstract public class AbstractAirthingsHandler extends BeaconBluetoothHandler {
                 default:
                     break;
             }
-        } catch (Exception e) {  // catch all to avoid scheduleWithFixedDelay being suppressed
+        } catch (Exception e) { // catch all to avoid scheduleWithFixedDelay being suppressed
             logger.warn("Failed to read Airthings device", e);
         }
     }

--- a/bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AbstractAirthingsHandler.java
+++ b/bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AbstractAirthingsHandler.java
@@ -111,23 +111,27 @@ abstract public class AbstractAirthingsHandler extends BeaconBluetoothHandler {
     }
 
     private synchronized void execute() {
-        ConnectionState connectionState = device.getConnectionState();
-        logger.debug("Device {} state is {}, serviceState {}, readState {}", address, connectionState, serviceState,
-                readState);
+        try {
+            ConnectionState connectionState = device.getConnectionState();
+            logger.debug("Device {} state is {}, serviceState {}, readState {}", address, connectionState, serviceState,
+                    readState);
 
-        switch (connectionState) {
-            case DISCOVERING:
-            case DISCOVERED:
-            case DISCONNECTED:
-                if (isTimeToRead()) {
-                    connect();
-                }
-                break;
-            case CONNECTED:
-                read();
-                break;
-            default:
-                break;
+            switch (connectionState) {
+                case DISCOVERING:
+                case DISCOVERED:
+                case DISCONNECTED:
+                    if (isTimeToRead()) {
+                        connect();
+                    }
+                    break;
+                case CONNECTED:
+                    read();
+                    break;
+                default:
+                    break;
+            }
+        } catch (Exception e) {  // catch all to avoid scheduleWithFixedDelay being suppressed
+            logger.warn("Failed to read Airthings device", e);
         }
     }
 

